Bottom: d0aaa100ec12ee02575ebd0e97e48c30f17bb3a4
Top:    df7fbb0889072024c08bcadd75d5738a9d5a785c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-09-30 22:44:50 +0100

Ensure "Gathering" mode is set after painting the screen.



---

diff --git a/src/draw.c b/src/draw.c
index 8322879..f710567 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -2711,4 +2711,6 @@ hid_expose_callback (HID * hid, BoxType * region, void *item)
   Output.fgGC = savefg;
   Output.bgGC = savebg;
   Output.pmGC = savepm;
+
+  Gathering = True;
 }
